lynx/common/actions/common_requirements.py

```diff
@@ -35,3 +36,11 @@ def can_be_stacked(scene: 'Scene', position: Vector, object_name: str) -> bool:
 				return True
 
 		return False
+
+	# TODO: Move, Push, Chop etc. should take argument of Direction type, not Vector type
```

Comment:
Wouldn't bother all the `enum` stuff. The requirement below is enough in my opinion.